app/models/tea_time.rb

```diff
@@ -222,6 +223,19 @@ def to_s
   end
 
   private
+    # schedule 1 OR 2 days before  depending on how far away it is
+    def queue_host_nudge_email
+      days_before = days_till_tea_time_start <= 2 ? 1 : 2
```

Comment:
@ankitshah811 this is the logic for nudge emails. 

If the tea time is less than or equal to 2 days away, we queue the nudge for 1 day before
if it's is more than 2 days away, we queue the nudge for 2 days before.

I think that should be good.